css/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: poppins;
+  src: url("https://fonts.googleapis.com/css?family=Poppins");
+}
+
+.header {
+  width: 375px;
+  height: 90px;
+  color: #6070ff;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Please make sure that the logo and menu icon is aligned to the center as present in the figma template. See the images below for more clarity 💡 This can be fixed by removing `margin-top` from `.material-icons` and adding `align-items: center` to `.header`
- [ ] Also, the menu icon you are using is a bit different from the one present in the Figma template. Can you please fix this as well ?😃

Your design | Given design
---| ---|
![image](https://github.com/jalal1379/Microverse_portflaio/assets/106119105/438815b1-501e-4f08-b481-5407abf9f052) | ![image](https://github.com/jalal1379/Microverse_portflaio/assets/106119105/78197b62-c625-4da0-be5d-617d660ab3ff)